amazon-eks-nodegroup.yaml

```diff
@@ -84,6 +84,11 @@ Parameters:
     Description: Maximum size of Node Group ASG.
     Default: 3
 
+  NodeVolumeSize:
+    Type: Number
+    Description: Node volume size
+    Default: 20
```

Comment:
20GB is very low to have as a default. For example, GKE uses 100GB.

Reply:
This is default value from AMI

Reply:
But that doesn't mean it should be 20 here. It can be set to any value.

Reply:
@micahhausler has previously mentioned that they want to stick to 20: https://github.com/awslabs/amazon-eks-ami/issues/13#issuecomment-408267434

Reply:
Seems a bit old fashioned but OK. At least with this PR we can change it now.